src/models/Flashcard.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const flashcardSchema = new mongoose.Schema({
+    topic: {
+        type: String,
+        unique: [true, 'This topic is already taken'],
```

Comment:
[Overly Restrictive Schema]
I assumed the topic was like the "subject" in school. So a topic wouldn't be unique to a flashcard. In other words, many flashcards may have the topic "Node".
Were we thinking of the topic as the title? If so we still don't want it to be unique cause different users should still be able to have flashcards with the same title probably.